benchmarks/dpe_bench.cc

```diff
@@ -0,0 +1,189 @@
+#include <iostream>
+#include <random>
+#include <map>
+#include <chrono>
+
+#include "hermes.h"
+#include "utils.h"
+#include "data_placement_engine.h"
+
+/* example usage: ./bin/dpe_bench -m -s 4096 */
+
+using namespace hermes;  // NOLINT(*)
+using std::chrono::time_point;
+const auto now = std::chrono::high_resolution_clock::now;
+
+enum class PlacementPolicy {
```

Comment:
Can't you use `hermes::api::PlacementPolicy` instead of redefining the policies here?